### PR TITLE
Improving docs to cover GSuite Google Groups

### PR DESCRIPTION
**Pull request by mirons-google**
This is a follow-up to #165 

* It updates the documentation to support the creation of a new service account
* How to enable domain-wide delegation for that service account
* How to pass these flags into `inventory_loader.py`

I'll follow-up with GCP installation specific instructions once @blueandgold is done re-working the pipelines at the same time I create a PR for enabling GSuite Google Group checking on GCP.

### CODE REVIEW

File: docs/inventory/README.md [hunk 1]
```diff
@@ -1,64 +1,48 @@
...
-that's ingestiable.
+1. Define a new table schema for the 'flattended' data you need to store.
+Each relevant field of data you retrieve from an API should correspond to a column in the table schema.
 
-Look at this PR for an [end-to-end example loading of a resource].
+2. Define a new table schema for the 'raw data you need to store.
+Additionally, we aim to store the raw API data as json to assist troubleshooting.
 
-Step-by-Step:
+Once you've completed these steps create a Pull Request ([example](https://github.com/GoogleCloudPlatform/forseti-security/pull/159)).
```
Comment (carise) [inline docs/inventory/README.md]: I would link to the official Github documentation on how to create a pull request, and not an actual pull request.

File: docs/prerequisites/README.md [hunk 2]
```diff
@@ -87,16 +94,18 @@ don't want to be tracked, the tracking pixel can be
...
 
   ```sh
   $ gcloud beta service-management enable deploymentmanager.googleapis.com
   ```
 
 ### Local installations
 See [PREREQUISITES-LOCALLY](/docs/prerequisites/PREREQUISITES-LOCALLY.md)
+
+## Finish installation
+Once completed continue with the the installation steps
```
Comment (carise) [inline docs/prerequisites/README.md]: Once you've completed the prerequisite steps

Comment (mirons-google) [inline docs/prerequisites/README.md]: done

File: docs/inventory/README.md (hunk 1 above)
Comment (mirons-google) [inline docs/inventory/README.md]: done

File: docs/inventory/README.md [hunk 3]
```diff
@@ -1,64 +1,48 @@
...
-The main steps are:
-
-1. Define the database table schema to store the resource data.
-
-2. Create a new pipeline worker to process the new table and the table fieldnames.
+It's straightforward to add a new resource to inventory for collection. This workflow is designed to be generic across resource types.
 
-3. Write a transformation to flatten the data returned by GCP APIs into a format
-that's ingestiable.
+1. Define a new table schema for the 'flattended' data you need to store.
```
Comment (mirons-google) [inline docs/inventory/README.md]: done

File: docs/inventory/README.md [hunk 4]
```diff
@@ -1,64 +1,48 @@
...
 
 ```sh
 $ forseti_inventory
 ```
 
-You can also use the convenience [dev_inventory.sh script](/scripts) to run
-forseti_inventory. Make a copy of dev_inventory.sh.sample
-as dev_inventory.sh, edit the script for the appropriate
-commandline flags, and invoke the script from the repo root to run inventory.
+You can also use the convenience [dev_inventory.sh script](/scripts) to run forseti_inventory. Make a copy of dev_inventory.sh.sampleas dev_inventory.sh, edit the script for the appropriate commandline flags, and invoke the script from the repo root to run inventory.
```
Comment (mirons-google) [inline docs/inventory/README.md]: done

File: docs/inventory/README.md [hunk 5]
```diff
@@ -1,64 +1,48 @@
 # Inventory
 This is the Inventory component of Forseti Security tool.
-
+  * [Pre-requisites](#pre-requisites)
```
Comment (mirons-google) [inline docs/inventory/README.md]: done

File: docs/installing/INSTALLING-LOCALLY.md [hunk 6]
```diff
@@ -17,6 +17,10 @@ $ workon forseti-security
 ## Get the source code
 Clone the repo, if you haven't already done so.
 
+```sh
+$ (forseti-secuirty): git clone https://github.com/GoogleCloudPlatform/forseti-security.git
```
Comment (mirons-google) [inline docs/installing/INSTALLING-LOCALLY.md]: done

File: docs/installing/INSTALLING-LOCALLY.md [hunk 7]
```diff
@@ -6,7 +6,7 @@
 * [Execution](#execution)
 
 ## Prerequisites
-Install or set up the Forseti Security [prerequisites](/docs/prerequisites/README.md).
+Complete the Forseti Security prequisites [README](/docs/prerequisites/README.md).
```
Comment (mirons-google) [inline docs/installing/INSTALLING-LOCALLY.md]: done

File: docs/common/SERVICE-ACCOUNT.md [hunk 8]
```diff
@@ -1,38 +1,48 @@
...
+
+## Create a service account for inventorying of GSuite Google Groups
+1. Create a custom service account in the [GCP console](https://console.cloud.google.com/iam-admin/serviceaccounts).
+ 1. Enable domain-wide delegation.
+1. Create and download the json key to your local environment.
+1. Note the domain-wide client-id, e.g. 1111111111111111111
+
+### Enable the service account in the GSuite admin control panel
+1. Visit the [advanced settings](https://admin.google.com/ManageOauthClients) portion of the control panel
+ 1. Specify the domain-wide client-id from above
```
Comment (mirons-google) [inline docs/common/SERVICE-ACCOUNT.md]: done

File: docs/common/SERVICE-ACCOUNT.md [hunk 9]
```diff
@@ -1,38 +1,48 @@
...
   --role=roles/browser \
   --role=roles/iam.securityReviewer \
   --role=roles/editor
 ```
+
+## Create a service account for inventorying of GSuite Google Groups
+1. Create a custom service account in the [GCP console](https://console.cloud.google.com/iam-admin/serviceaccounts).
+ 1. Enable domain-wide delegation.
+1. Create and download the json key to your local environment.
+1. Note the domain-wide client-id, e.g. 1111111111111111111
```
Comment (mirons-google) [inline docs/common/SERVICE-ACCOUNT.md]: done

File: docs/common/SERVICE-ACCOUNT.md [hunk 10]
```diff
@@ -1,38 +1,48 @@
-# Service account
-In order to run Forseti Security, you must add a service account
-to your **organization-level** IAM policy.
+# Service accounts
+There are two service accounts that need created:
```
Comment (mirons-google) [inline docs/common/SERVICE-ACCOUNT.md]: done

**Comment by mirons-google** [comment]
Thanks for the review. I implemented your requested changes. PTAL.

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR author is a googler at the time of this PR was created.

**Pull request merged**